dot/parachain/statement-distribution/cluster_tracker_test.go

```diff
@@ -363,3 +365,386 @@ func TestClusterTracker_send_statements(t *testing.T) {
 		)
 	})
 }
+
+func TestClusterTracker_pendingStatementsFor(t *testing.T) {
+	group := []parachaintypes.ValidatorIndex{5, 200, 24, 146}
+	secondingLimit := uint(1)
+	hashA := parachaintypes.CandidateHash{Value: common.Hash{0x1}}
+	hashB := parachaintypes.CandidateHash{Value: common.Hash{0x2}}
+
+	// Test that the `pending_statements` are set whenever we receive a fresh statement.
+	//
+	// Also test that pending statements are sorted, with `Seconded` statements in the front.
+	t.Run("pending_statements_set_when_receiving_fresh_statements", func(t *testing.T) {
+		tracker := newClusterTracker(group, secondingLimit)
```

Comment:
add `t.Parallel()` to all the test cases
```suggestion
	t.Parallel()
		tracker := newClusterTracker(group, secondingLimit)
```